public/components/ContextMenu/CreateAnomalyDetector/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {
+  EuiLink,
+  EuiText,
+  EuiHorizontalRule,
+  EuiSpacer,
+  EuiPanel,
+  EuiIcon,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiButton,
+} from '@elastic/eui';
+import { useField, useFormikContext } from 'formik';
+import Notifications from '../Notifications';
+import FormikWrapper from '../../../utils/contextMenu/FormikWrapper';
+import './styles.scss';
+import { toMountPoint } from '../../../../../../src/plugins/opensearch_dashboards_react/public';
+
+export const CreateAnomalyDetector = (props) => {
+  const { overlays, closeMenu } = props;
+  const { values } = useFormikContext();
+  const [name] = useField('name');
+
+  const onOpenAdvanced = () => {
+    // Prepare advanced flyout with new formik provider of current values
+    const getFormikOptions = () => ({
+      initialValues: values,
+      onSubmit: (values) => {
+        console.log(values);
+      },
```

Comment:
is this ever called?

Reply:
this file is not is being used anymore. sorry forgot to clean this file